expansion/src/test/java/no/unit/nva/expansion/model/ExpandedResourceUpdateTest.java

```diff
@@ -54,7 +54,7 @@ void expandedResourceShouldHaveTypePublicationInheritingTheTypeFromThePublicatio
         var publication = PublicationGenerator.randomPublication();
         var expandedResource = fromPublication(publication);
         var json = objectMapper.readTree(expandedResource.toJsonString());
-        assertThat(json.get(ExpandedDatabaseEntry.TYPE_FIELD).textValue(), is(equalTo("Publication")));
+        assertThat(json.get("type").textValue(), is(equalTo("Publication")));
```

Comment:
removed the static field, because it has no other use but to reference a hardcoded value that lies in the `@JsonTypeInfo` annotations.